server/chat-plugins/help.ts

```diff
@@ -0,0 +1,463 @@
+/**
+ * PS Help room auto-response plugin.
+ * Uses Regex to match room frequently asked question (RFAQ) entries,
+ * and replies if a match is found.
+ * Supports configuration.
+ * Written by mia-pi.
+ */
+
+import {FS} from '../../lib/fs';
+import {Utils} from '../../lib/utils';
+import {LogViewer} from './chatlog';
+import {ROOMFAQ_FILE} from './room-faqs';
+
+const PATH = 'config/chat-plugins/help.json';
+// 6: filters out conveniently short aliases
+const MINIMUM_LENGTH = 6;
+const HELP_ROOM = Config.helpFilterRoom ? Rooms.get(Config.helpFilterRoom) : Rooms.get('help');
```

Comment:
This is a persistent reference to the ``Room`` instance that would prevent it from GC if the room is deleted.
This is also the reason why this plugin cannot be required from tests.

It'd Just reference the ``Config`` global from the relevant functions.